ammo-be/src/webSocket/index.ts

```diff
@@ -3,7 +3,15 @@ import http from 'http';
 
 let ioServer: io.Server;
 
-let isRecording = false;
+// Bullets sub statuts
+let isSubbedToBullets = false;
+const getIsSubbedToBullets = (): boolean => isSubbedToBullets;
+
+// Disconnect callbacks. Can be useful for testing
+const disconnectCbks: { (): void }[] = [];
```

Comment:
It's the typing for a function returning void and taking no argument. Don't know how to write it simpler. TS won't let me use 'Function' because it's too generic

Reply:
Oh, it's an array of functions returning void ? I think taking it as a named typed would help readability

Reply:
Like this : 

```js

type FunctionReturningVoid = () => void;

// Disconnect callbacks. Can be useful for testing
const disconnectCbks: FunctionReturningVoid[] = [];
```